packages/lexicon/src/validators/formats.ts

```diff
@@ -1,24 +1,22 @@
-import { isValidISODateString } from 'iso-datestring-validator'
 import { CID } from 'multiformats/cid'
 import { ValidationResult, ValidationError } from '../types'
 import {
   ensureValidDid,
   ensureValidHandle,
   ensureValidNsid,
   ensureValidAtUri,
+  ensureValidDatetime,
 } from '@atproto/syntax'
 import { validateLanguage } from '@atproto/common-web'
 
 export function datetime(path: string, value: string): ValidationResult {
   try {
-    if (!isValidISODateString(value)) {
-      throw new Error()
-    }
+    ensureValidDatetime(value)
```

Comment:
yeah this will happen everywhere - on new record creations & when validating existing records

I think we leave this code path as-is for now, and I can add in the more rigorous validation into the pds indexing path